crates/rspack_plugin_css/tests/fixtures/custom/css_auto/expected/main.css

```diff
@@ -0,0 +1,6 @@
+.style {
```

Comment:
avoid to add tests on rust side, we lack of human maintenance and js side test is good enough in most of cases